proto/storage/proto.lock

```diff
@@ -11541,7 +11547,7 @@
               },
               {
                 "id": 5,
-                "name": "alert_notifier",
+                "name": "data_collection_rule_config",
```

Comment:
Would probably be best if you made this change in the parent PR before merging it if this is the desired name.

Reply:
Build broken, the other branch reported the check-generated-files as passing, but actually it failed.